### ISSUE TITLE
NVDA doesn't announce custom snackbar in IE 11

### ISSUE THREAD

**Issue by diminutivesloop** [angular/material2#12270]
#### Bug, feature request, or proposal:
Bug

#### What is the expected behavior?
Snackbar messages should be announced via screen reader.

#### What is the current behavior?
When activating a custom snackbar in IE 11 NVDA does not announce snackbar text.

#### What are the steps to reproduce?
https://stackblitz.com/edit/angular-gbkaav

#### Which versions of Angular, Material, OS, TypeScript, browsers are affected?
Angular 6.0.9
Material 6.4.0
IE 11
NVDA 2017.4

**Comment by josephperrott** [comment]
Closing, as this is working as expected as the ARIA announcer announces the message provided in the `MatSnackBarConfig`.  This is done in the [`open()`](https://github.com/angular/material2/blob/master/src/lib/snack-bar/snack-bar.ts#L115) method automatically as a convenience for the user.

**Comment by diminutivesloop** [comment]
@josephperrott Ok well I apparently missed the `MatSnackBarConfig.announcementMessage` option. However even though that option fixes the behavior in IE 11 it also causes the message to be announced twice in Chrome. That certainly doesn't sound like expected behavior to me.
